test/e2e/e2e_test.go

```diff
@@ -127,6 +143,62 @@ var _ = Describe("License Compliance Checker", func() {
 		})
 	})
 
+	Context("modules", func() {
+		It("should check a project's modules", func() {
```

Comment:
I would add a test showing how it fails when `GO111MODULE` is not provided

Reply:
Can we also add a test showing a usage without the `check-go-modules` option?

Reply:
testing when `GO111MODULE` is not set is a little more awkward, since that behaviour depends on whether the tests are being run from inside the GOPATH